cmd/config/cnv-metrics.yml

```diff
@@ -0,0 +1,124 @@
+  instant: true
+
+- query: avg(avg_over_time(kubevirt_memory_delta_from_requested_bytes[{{.elapsed}}:])) by (container)
+  metricName: avg-infra-container-request-usage-delta
+  instant: true
+
+- query: max(max_over_time(kubevirt_memory_delta_from_requested_bytes[{{.elapsed}}:])) by (container)
+  metricName: max-infra-container-request-usage-delta
+  instant: true
+
+- query: avg_over_time(label_replace(sum(container_memory_rss{pod=~"virt-launcher.*"}) by (pod), "vmi", "$1", "pod", "virt-launcher-(.*)-[^-]+$")[{{.elapsed}}:]) 
+  metricName: avg-mem-usage-by-vmi
+  instant: true
+
+- query: avg_over_time(label_replace(sum(rate(container_cpu_usage_seconds_total{pod=~"virt-launcher.*"}[2m])) by (pod),"vmi", "$1", "pod", "virt-launcher-(.*)-[^-]+$")[{{.elapsed}}:])
+  metricName: avg-cpu-usage-by-vmi
+  instant: true
+
+## storage related metrics
+- query: avg(avg_over_time(rate(kubevirt_vmi_storage_iops_read_total[60s])[{{.elapsed}}:])) by (name) 
+  metricName: avg-vmi-read-iops
+  instant: true
+
+- query: avg(avg_over_time(rate(kubevirt_vmi_storage_iops_write_total[60s])[{{.elapsed}}:])) by (name)
+  metricName: avg-vmi-write-iops
+  instant: true
+
+- query: avg(avg_over_time(rate(kubevirt_vmi_storage_read_traffic_bytes_total[60s])[{{.elapsed}}:])) by (name)
+  metricName: avg-vmi-storage-read-traffic
+  instant: true
+
+- query: avg(avg_over_time((rate(kubevirt_vmi_storage_write_traffic_bytes_total[60s])[{{.elapsed}}:]))) by (name) 
+  metricName: avg-vmi-storage-write-traffic
+  instant: true
+
+## network related metrics
+- query: avg(avg_over_time(rate(kubevirt_vmi_network_receive_bytes_total[60s])[{{.elapsed}}:])) by (name)
+  metricName: avg-vmi-traffic-received
+  instant: true
+
+- query: avg(avg_over_time(rate(kubevirt_vmi_network_transmit_bytes_total[60s])[{{.elapsed}}:])) by (name) 
+  metricName: avg-vmi-traffic-transmitted
+  instant: true
+
+## Virt API reated metrics
+- query: avg_over_time(sum(rate(apiserver_request_total{group=~".*kubevirt.*", verb=~"LIST|GET"}[1m]))[{{.elapsed}}:]) 
+  metricName: avg-virt-api-read-request-rate
+  instant: true
+
+- query: avg_over_time(sum(rate(apiserver_request_total{group=~".*kubevirt.*", verb=~"POST|PUT|PATCH|DELETE"}[1m]))[{{.elapsed}}:]) 
+  metricName: avg-virt-api-write-request-rate
+  instant: true
+
+- query: avg_over_time(sum(rate(apiserver_request_total{group=~".*kubevirt.*", code!~"2.."}[1m]))[{{.elapsed}}:]) 
+  metricName: avg-virt-api-error-rate
+  instant: true
+
+## vmi migration reated metrics
+- query: sum(kubevirt_vmi_migration_succeeded)
+  metricName: migration-succeeded
+  instant: true
+
+- query: sum(kubevirt_vmi_migrations_in_scheduling_phase)
+  metricName: migration-in-scheduling-phase
+  instant: true
+
+- query: sum(kubevirt_vmi_migrations_in_running_phase)
+  metricName: migration-in-running-phase
+  instant: true
+
+- query: sum(kubevirt_vmi_migrations_in_pending_phase)
+  metricName: migration-in-pending-phase
+  instant: true
```

Comment:
I think we should avoid migration metrics for the moment, as they are only relevant in migration in scenarios running migrations, which is not the case. @ygalblum has been working on some of them, but I think he's not currently using kube-burner's indexing capabilities.


Also, if we are adding these metrics, they shouldn't be instant, as migrations is an operation that happens during the whole workoad runtime and pulling the last datapoint won't probably return anything relevant

Reply:
what about removing the instant and keep them for now? At some point, we will add some migration tests.